cmd/opentelemetry/app/exporter/elasticsearchexporter/esmodeltranslator/modeltranslator_test.go

```diff
@@ -274,7 +269,18 @@ func addSpan(traces pdata.Traces, name string, traceID pdata.TraceID, spanID pda
 	span.SetName(name)
 	span.SetTraceID(traceID)
 	span.SetSpanID(spanID)
+	span.SetParentSpanID(spanID)
 	span.SetStartTime(pdata.TimestampUnixNano(time.Now().UnixNano()))
 	span.SetEndTime(pdata.TimestampUnixNano(time.Now().UnixNano()))
 	return span
 }
+
+func Byte16ArrayFromString(s string) (result [16]byte) {
```

Comment:
Think I should move these functions not to duplicate code. Would be great if you could help to find their place

Reply:
How many callers are there? If there are only two, change them to use something like `pdata.NewTraceID([16]byte{1, 2, 3, 4}` instead of a string. I don't think being a string is part of the test.